test.py

```diff
@@ -68,6 +68,9 @@ def test(test_name):
 else:
     @pytest.mark.parametrize("test_name", tests)
     def test_node_wat64(test_name):
+        if test_name == "extern":

```

Comment:
It'd be better if this was a part of the test file itself, but this is a matter for a different PR for sure